### PR TITLE
Farq panay change

### PR DESCRIPTION
**Pull request by FinleyInfoTechSys**

### CODE REVIEW

File: events/MTG_USA.txt [hunk 1]
```diff
@@ -4396,36 +4396,36 @@ country_event = {
 		hidden_effect = { JAP = { country_event = { id = panay_incident.3 days = 2 } } } #Delay added so events are spaced out - historically it took months to get JAP to agree to compensation, but that'd be a bit hard to follow in-game, so timelines compressed.
 		add_opinion_modifier = { target = JAP modifier = panay_inc_opinion_modifier }
 	}
-	option = { #War
```
Comment (myzael) [inline events/MTG_USA.txt:4399]: It will work just fine by removing the option. But I was talking with voigt and he raised a valid issue that USA going for an interventionist path should probably be allowed to declare this war. They're hit with heavy penalties for it anyway.

I would suggest you tried to change it so that the option is not removed, but only visible when USA is not on undisturbed isolation. You can do that by adding `trigger` block in the `option` block for this event.

File: new_changelog.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Removed war option from Panay incident in mtg_usa.txt
```
Comment (myzael) [inline new_changelog.txt:1]: From my side, you don't need to add a separate changelog. I have a script that takes commit messages and compiles a changelog from them. If you don't want to include a given commit message in the changelog, just start it with "[NC]".

**Review by myzael — COMMENTED** [review]
Good work as your first time working with modding and with git!

For keeping your local files up to date with the main mod, read up on how git pull works.

File: events/MTG_USA.txt (hunk 1 above)
Comment (FinleyInfoTechSys) [reply events/MTG_USA.txt:4399]: Let me get the trigger in and test.  the alex/master should be the origin and my fork upstream correct?

File: new_changelog.txt (hunk 2 above)
Comment (FinleyInfoTechSys) [reply new_changelog.txt:1]: noted

**Comment by myzael** [comment]
Closing this one as another PR supersedes it.
